### ISSUE TITLE
ドラえもん どら焼きパックンゲーム (Doraemon dorayaki pakkun game) - Glitched Japanese text

### ISSUE THREAD

**Issue by SeruranBlue** [ruffle-rs/ruffle#10014]
### Describe the bug

Link to SWF:https://web.archive.org/web/20050118054619/http://dora-world.com/game/mt_dor09.swf

Several Japanese Text is broken. (maybe text encoding problem.)

![a](https://user-images.githubusercontent.com/121824800/224536383-7644d77b-3112-45bc-bff1-54d5e2f95be1.png)

### Expected behavior

![b](https://user-images.githubusercontent.com/121824800/224536388-0753b3c6-c6e1-4d93-a954-e18c937cb134.png)

### Affected platform

Desktop app

### Operating system

win 10

### Additional information

ruffle 2023-03-12 version.

Aside:
This game is an official Doraemon Flash game. It released around 2002, and deleted from an website around 2005.

**Comment by n0samu** [comment]
Yes, it is an encoding issue. The same issue as the examples from #8390.
